site/app/models/User.php

```diff
@@ -485,6 +495,15 @@ public static function validateUserData($field, $data): bool {
                 // validate email address against email RFCs
                 $validator = new EmailValidator();
                 return $validator->isValid($data, new RFCValidation());
+            case 'user_email_secondary':
```

Comment:
Does this do the same thing as the above check? If so you can do

```php
case 'user_email':
case 'user_email_secondary':

...

return ...

```

Reply:
Yes it should do the exact same thing. I didn't think of doing it that way. I will change it to that.